.chloggen/fix-opamp-bridge-missing-config.yaml

```diff
@@ -0,0 +1,16 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: bug_fix
+
+# The name of the component, or a single word describing the area of concern, (e.g. collector, target allocator, auto-instrumentation, opamp, github action)
+component: opamp
+
+# A brief description of the change. Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Fixes the type for AnyConfig in the collector to allow for proper marshalling
```

Comment:
This description is now out of date I think, we're now just fixing a marshalling bug in the bridge.